### ISSUE TITLE
Cannot use

### ISSUE THREAD

**Issue by warlockuk** [Xananax/calibre-serve#1]
sudo npm install -g calibre-serve

go to test library folder.

calibre-serve foldernameabovedatabasefolder
/usr/local/lib/node_modules/calibre-serve/cli.js:4
const {name,version} = pkg;
         ^
calibre-serve -h
/usr/local/lib/node_modules/calibre-serve/cli.js:4
const {name,version} = pkg;
         ^

Is there a specific node version required or other dependencies?  None of the code seems to work.

**Comment by Xananax** [comment]
As far as I can see, your problem is with the `const` keyword. Are you sure you're using an up to date version of node?

**Comment by warlockuk** [comment]
I'm using 4.2.6; I'm presuming that using const requires node 6 rather than the latest 4.
const is fine.  This bug report actually omitted some spaces - the ^ should be under {name,version}

I've got a feeling that it's not pulling the pkg json file in correctly, maybe some kind of file error for that.

**Comment by warlockuk** [comment]
Update: switched node version to 6.6.0; it seems to work.

Might be worth noting node6 as a dependency instead of node4 LTS.

**Comment by Xananax** [comment]
Yes, I should. I forget that there are several versions of node in circulation that people use.

**Comment by warlockuk** [comment]
I'm trying to run it on a linux server; the versions of node and npm available in the package manager is 4.2.6 + 3.5.2.

These don't work.

I updated to node 6.6.0 and the version of npm that comes with, but there's a version conflict with the version of something sqllite3 uses; I tried a reinstall and had npm fall on its arse.

I think I need to manually remove everything + manually install again.  Hopefuly there'll be no conflicts with my own stuff :)

**Comment by Xananax** [comment]
If you're willing to wait a week or so, I'll add in some babel/webpack to compile to ES5

**Comment by warlockuk** [comment]
Yeah, I'm in absolutely no rush.  I've only just started messing with your node app, but it looks excellent so far!
